emu/src/mem.rs

```diff
@@ -51,16 +51,17 @@ impl Mem {
     }
     pub fn write_u8(&mut self, address: u32, value: u32) {
         match address & 0xFFFFFF {
-            0..=0x3FFFFF => {},
+            0..=0x9FFFFF => {},
             0xA00000..=0xA03FFF => self.z80.write_byte(address, value),
             0xA04000..=0xA0FFFF => {},
-            0xA10000..=0xA1001F => self.io.write_byte(address, value),
-            0xA11100..=0xA112FF => self.z80.ctrl_write(address, value),
+            0xA10000..=0xA10FFF => self.io.write_byte(address & 0xFFF01F, value),
+            0xA11000..=0xA110FF => {},
+            0xA11100..=0xA11FFF => self.z80.ctrl_write(address, value),
             0xC00000..=0xDFFFFF => {/* PSG goes here? */},
             0xFF0000..=0xFFFFFF => {
                 self.ram[address as usize & 0xFFFF] = value as u8;
             },
-            _ => todo!("write byte {:X} {:X}", address, value),
+            _ => {}, //todo!("write byte {:X} {:X}", address, value),
```

Comment:
this should still work now without being commented out, right?

Reply:
Nope.

Reply:
do you know which addresses these are? can we put them in the match statement and add a comment about them instead? I would like to keep this 'catch all' section to panic on unimplemented things still

Reply:
I forget honestly, since I was hitting a TON of them, so I just decided to comment that out lol